### DISCUSSION TITLE
司法院智慧客服小幫手

### DISCUSSION THREAD

**Discussion by zixunn** [1101-GS4719-C/Python-Community-GS4719-C#322] [General]
普通民眾若遇上司法糾紛，有時不太了解救濟管道的人就會上網找資料，但有時網路上的資料似是而非或是太過繁雜，這時候有一個司法院智慧客服小幫手感覺就方便很多。
[https://talk.ltn.com.tw/article/breakingnews/3633591](https://talk.ltn.com.tw/article/breakingnews/3633591)

**Comment by cody910120** [comment]
我只希望這些客服的回答不要太官方就好

**Comment by yt7l0** [comment]
用抓關鍵字的方式直接回覆相關法律條文或文件，其實很方便

**Comment by YuHengChen54** [comment]
這樣一來就可以更方便的查法律相關的東西了
